python/pyspark/sql/tests/connect/test_parity_pandas_udf_scalar.py

```diff
@@ -21,17 +21,6 @@
 
 
 class PandasUDFScalarParityTests(ScalarPandasUDFTestsMixin, ReusedConnectTestCase):
-    def test_nondeterministic_vectorized_udf_in_aggregate(self):
```

Comment:
this overridden method is no longer needed, since we support quite_test in connect